packages/adapter/package.json

```diff
@@ -28,8 +28,8 @@
     "@metamask/snaps-ui": "0.32.2",
     "@shapeshiftoss/hdwallet-core": "^1.29.0",
     "@shapeshiftoss/logger": "^1.1.3",
-    "@shapeshiftoss/metamask-snaps": "workspace:^",
-    "@shapeshiftoss/metamask-snaps-types": "workspace:^"
+    "@shapeshiftoss/metamask-snaps": "^1.0.0-alpha.11",
+    "@shapeshiftoss/metamask-snaps-types": "^1.0.0-alpha.11"
```

Comment:
Related to the lerna build magic mentioned above - running `lerna version` / `lerna version` will automagically update imports across all packages. Using workspaces works fine when running the whole thing locally as a workspace i.e when devving packages themselves, but doesn't work for consuming packages from other packages, since they reference `workspace:` as a prefix, which is obviously invalid in actual real install scenario